src/client/modules/ExportWins/Review/Layout.jsx

```diff
@@ -62,14 +68,87 @@ const Title = styled(H1)({
   marginBottom: SPACING.SCALE_5,
 })
 
+const CookieBannerBackground = styled.div({
+  gridRow: 'cookie-banner',
+  gridColumn: '1 / -1',
+  background: '#f3f2f1',
+})
+
+const CookieBanner = styled.div({
+  gridArea: 'cookie-banner',
+  paddingTop: SPACING.SCALE_3,
+})
+
+const CookieConsentConfirmation = () => (
+  <RecentTaskResult name="save cookie preference" id="cookieConsent">
+    {(consent) =>
+      consent && (
+        <StyledStatusMessage colour={GREEN}>
+          You've {consent === 'granted' ? 'accepted' : 'rejected'} additional
+          cookies. You can change your cookie settings at any time.
+        </StyledStatusMessage>
+      )
+    }
+  </RecentTaskResult>
+)
+
 const Layout = ({ children, title, supertitle, headingContent }) => (
   <Grid>
+    <CookieBannerBackground />
+    <Resource name="load cookie preference" id="cookieConsent">
+      {(persistedConsent) => (
+        <RecentTaskResult name="save cookie preference" id="cookieConsent">
+          {(updatedConsent) =>
+            !persistedConsent &&
+            !updatedConsent && (
+              <CookieBanner>
+                <H2>Cookies</H2>
+                <p>
+                  We’d like to use analytics cookies so we can understand how
+                  you use the Design System and make improvements.
+                </p>
+                <p>
+                  We also use essential cookies to remember if you’ve accepted
+                  analytics cookies.
+                </p>
+                <FormActions>
+                  <Task>
+                    {(getTask) => {
+                      const setPreference = (value) =>
+                        getTask(
+                          'save cookie preference',
+                          'cookieConsent'
+                        ).start({
+                          payload: value,
+                          onSuccessDispatch: 'FORM__RESOLVED',
+                        })
+                      return (
+                        <>
+                          <Button onClick={() => setPreference('granted')}>
+                            Accept analytics cookies
+                          </Button>
+                          <Button onClick={() => setPreference('denied')}>
+                            Reject analytics cookies
+                          </Button>
+                        </>
+                      )
+                    }}
+                  </Task>
+                  <Link href="/exportwins/review/cookies">View cookies</Link>
+                </FormActions>
+              </CookieBanner>
+            )
+          }
+        </RecentTaskResult>
+      )}
+    </Resource>
     <MainBarBackground />
     <MainBar>Department for Business & Trade</MainBar>
```

Comment:
This isn't relevant to the PR but the `&` should only be used as part of the logo. For all other cases it should be written with the word `and`
```suggestion
    <MainBar>Department for Business and Trade</MainBar>
```